pubpub/platform
# Convert form bookmark button to copy link button

## Issue(s) Resolved

Resolves #715

## High-level Explanation of PR


This PR changes the previously non-functional bookmark button on the form fill page into a "copy link" button that copies the form URL for easy sharing.

## Test Plan

- Visit the a form fill page.
- Click the copy link.
- Confirm that the page URL was copied.
